### PR TITLE
Add column to metadata and change year aggregation value

### PR DESCRIPTION
**Pull request by albrja**
### Adds number of rows column and changes year aggregation value
- *Category*: Post-processing
- *JIRA issue*: [MIC-4670](https://jira.ihme.washington.edu/browse/MIC-4670)
- *Research reference*: 

### Changes and notes
-Adds column to metadata
-updates year aggregation value for quality of life

### CODE REVIEW

File: src/vivarium_census_prl_synth_pop/utilities.py [hunk 1]
```diff
@@ -576,7 +576,7 @@ def record_metadata_proportions(final_output_dir: Path) -> None:
             .sum()
             .reset_index()
         )
-        state_aggregated_metadata["year"] = "all"
+        state_aggregated_metadata["year"] = 3000
```
Comment (mattkappel) [inline src/vivarium_census_prl_synth_pop/utilities.py:579]: Why 3000?

File: src/vivarium_census_prl_synth_pop/utilities.py [hunk 2]
```diff
@@ -576,7 +576,8 @@ def record_metadata_proportions(final_output_dir: Path) -> None:
             .sum()
             .reset_index()
         )
-        state_aggregated_metadata["year"] = "all"
+        # TODO: import from Pseudopeople once released
```
Comment (mattkappel) [inline src/vivarium_census_prl_synth_pop/utilities.py:579]: Is there a Jira ticket for this?

**Review by mattkappel — APPROVED** [review]

File: src/vivarium_census_prl_synth_pop/utilities.py [hunk 3]
```diff
@@ -609,9 +610,7 @@ def record_metadata_proportions(final_output_dir: Path) -> None:
             melted_metadata["value"] / melted_metadata["number_of_rows"]
         )
         # Drop unnecessary columns
-        melted_metadata = melted_metadata.drop(
-            columns=["variable", "value", "number_of_rows"]
-        )
+        melted_metadata = melted_metadata.drop(columns=["variable", "value"])
```
Comment (stevebachmeier) [inline src/vivarium_census_prl_synth_pop/utilities.py:613]: Why are you dropping the number of rows?

File: src/vivarium_census_prl_synth_pop/constants/metadata.py [hunk 4]
```diff
@@ -295,3 +295,6 @@ class DatasetNames:
     "date_of_birth": "copy_date_of_birth",
     "has_ssn": "copy_ssn",
 }
+
+
+YEAR_AGGREGATION_VALUE = 3000
```
Comment (stevebachmeier) [inline src/vivarium_census_prl_synth_pop/constants/metadata.py:300]: What is this?

File: src/vivarium_census_prl_synth_pop/utilities.py (hunk 3 above)
Comment (albrja) [reply src/vivarium_census_prl_synth_pop/utilities.py:613]: Im NOT dropping the number of rows so we can weight the proportions for SSA later on in pseudopeople.

File: src/vivarium_census_prl_synth_pop/utilities.py (hunk 2 above)
Comment (albrja) [reply src/vivarium_census_prl_synth_pop/utilities.py:579]: Yes

File: src/vivarium_census_prl_synth_pop/constants/metadata.py (hunk 4 above)
Comment (stevebachmeier) [reply src/vivarium_census_prl_synth_pop/constants/metadata.py:300]: Maybe -1 would be better than?

**Review by stevebachmeier — APPROVED** [review]

**Review by patricktnast — APPROVED** [review]

File: src/vivarium_census_prl_synth_pop/constants/metadata.py (hunk 4 above)
Comment (rmudambi) [reply src/vivarium_census_prl_synth_pop/constants/metadata.py:300]: Also maybe this variable should be changed to make it clear that this is a nonsense value.

**Review by rmudambi — APPROVED** [review]

**Pull request merged**